packages/@lwc/test-runner/src/runner.js

```diff
@@ -60,6 +66,8 @@ export async function main() {
           files: testGlobPattern,
         },
       ],
+      browsers,
+      concurrency: 1,
```

Comment:
This is unfortunate, but if it resolves the problems people are having, it is a great medium-term solution. There's a deeper bug that needs to be addressed, but we can chat separately about whether we're going to take that on soon or just defer to a later date.